demisto_sdk/commands/validate/validators/RN_validators/RN113_is_valid_content_type_header.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import (
+    CONTENT_TYPE_SECTION_REGEX,
+    RN_HEADER_BY_FILE_TYPE,
+    GitStatuses,
+)
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.tools import (
+    filter_nones,
+)
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+class IsValidContentTypeHeaderValidator(BaseValidator[ContentTypes]):
+    error_code = "RN113"
+    description = "Validate that the rn's first level headers are valid content types."
```

Comment:
is it standard to put rn instead of "release notes" in our docs? if not we should spell it out